src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -83,6 +84,7 @@ public OpenApiDocument GetSwagger(string documentName, string host = null, strin
             var applicableApiDescriptions = _apiDescriptionsProvider.ApiDescriptionGroups.Items
                 .SelectMany(group => group.Items)
                 .Where(apiDesc => !(_options.IgnoreObsoleteActions && apiDesc.CustomAttributes().OfType<ObsoleteAttribute>().Any()))
+                .Where(apiDesc => !apiDesc.CustomAttributes().OfType<SwaggerIgnoreAttribute>().Any())
```

Comment:
Could this be `!HasAttribute<T>()`? I just saw it in the previous file and it looks terser than this.

Reply:
The behavior of this particular extension appears to be provided by the builtin PropertyInfo.IsDefined(Type). I've removed this extension on PropertyInfo, and added extensions with the same name on ApiDescription and ApiParameterDescription, as in this case the extension is required to handle e.g. property and parameter sourced API parameters.

It also occurred to me that the ApiExplorer provided models include the instantiated attributes as metadata, however I am unclear how this might affect synthetically generated instances in unit tests.